main.tf

```diff
@@ -64,14 +64,14 @@ module "obs_testnet_vpc" {
   deploy_rds_db                          = true
   xchain_settings = {
     enabled      = true
-    docker_image = var.xchain_indexer_docker_image
+    docker_image = "omniops/xchain-indexer:master"
     config       = "testnet"
     omni_config = {
       omni_rpc = local.omni_testnet_rpc
     }
   }
   blockscout_settings = {
-    blockscout_docker_image = var.testnet_blockscout_docker_image
+    blockscout_docker_image = "omniops/blockscout:master"
     rpc_address             = "http://testnet-sentry-explorer.omni.network:8545"
```

Comment:
let's use the local var here. we can set the local var to this url, since we've allocated a sentry just for the explorer.

Reply:
Done